mirrord-agent/src/main.rs

```diff
@@ -238,6 +253,46 @@ impl ClientConnectionHandler {
                     .send(DaemonMessage::GetEnvVarsResponse(env_vars_result))
                     .await?
             }
+            ClientMessage::GetAddrInfoRequest(GetAddrInfoRequest {
+                node,
+                service,
+                hints,
+            }) => {
```

Comment:
I prefer this to be in a function as this enum is huge and should have 1-2 lines per variant